components/server/src/external/routes/source.py

```diff
@@ -142,17 +142,16 @@ def post_source_parameter(source_uuid: SourceId, parameter_key: str, database: D
     if old_value == new_value:
         return dict(ok=True)  # Nothing to do
     edit_scope = cast(EditScope, dict(bottle.request.json).get("edit_scope", "source"))
-    changed_ids = change_source_parameter(data, parameter_key, old_value, new_value, edit_scope)
+    changed_ids_and_source_ids = change_source_parameter(data, parameter_key, old_value, new_value, edit_scope)
```

Comment:
```suggestion
    changed_ids, changed_source_ids = change_source_parameter(data, parameter_key, old_value, new_value, edit_scope)
```

Reply:
This will lead to the linter complaining about too many local variables. I think the alternative might be to split this function into two.

Reply:
Yeah, making the code worse to prevent a linter warning is putting the horse in front of the cart, so splitting the method or suppressing the warning are both better options than not destructuring the tuple IMO.